.travis.yml

```diff
@@ -9,6 +9,7 @@ rvm:
   - 2.5
   - 2.6
   - 2.7
+  - 3.0
```

Comment:
In order for this YAML to pick the 3.0 version, it may be even better to quote this value, that is, to say `"3.0"`. YAML encoding a Float to a String becomes 3.0 -> "3".